src/main/java/com/vmware/vipclient/i18n/messages/service/StringService.java

```diff
@@ -102,4 +116,124 @@ public boolean isStringAvailable(MessagesDTO dto) {
         }
         return r;
     }
+
+    public ComponentService.TranslationsDTO getMultiVersionKeyTranslations(String version) {
+        Iterator<Locale> fallbackLocalesIter = LocaleUtility.getFallbackLocales().iterator();
+        return this.getMultiVersionKeyTranslations(version, fallbackLocalesIter);
+    }
+
+    public ComponentService.TranslationsDTO getMultiVersionKeyTranslations(String version, Iterator<Locale> fallbackLocalesIter) {
+        return this.getMultiVersionKeyCacheItem(version, fallbackLocalesIter);
+    }
```

Comment:
i would combine the caller functions with the called functions if there's no logic codes.